### DISCUSSION TITLE
AWS S3 Image Upload  - Process clarification

### DISCUSSION THREAD

**Discussion by MarvingMoreton** [vercel/community#1086] [Help] [answered]
Hello there!

I am new to the AWS world, and I am trying to host my images to AWS S3's bucket.

I am at the point 4 - 5 of Vercel's tutorial: https://vercel.com/templates/next.js/aws-s3-image-upload-nextjs

However, when I run "cdk bootstrap" I get the following error: Specify an environment name like 'aws://123456789012/us-east-1', or run in a directory with 'cdk.json'.

<img width="900" alt="Screenshot 2022-11-24 at 2 41 03 PM" src="https://user-images.githubusercontent.com/96314887/203855290-8a299df0-930e-453e-8a5b-03210b532559.png">

**Comment by swarnava** [comment]
Hi,

It indicates that you don't have environment variable setup. Have setup these [variables](https://github.com/vercel/examples/blob/main/solutions/aws-s3-image-upload/.env.example) in `.env.local`?

**Comment by MarvingMoreton** [comment]
Hello

Yes I did. The access_key & secret_key are of the root user right?

The thing that I do not have in comparison to the repo example is the
"cdk.json"

Best,

**Reply by swarnava** [reply]
Perhaps, existing GitHub repository will give some pointer: https://github.com/search?q=cdk.json+language%3AJSON&type=code&l=JSON ?

**Comment by MarvingMoreton** [comment, marked as answer]
Yeah... Unfortunately the provided process seems not clear to me

I think I'll give this one a try:
https://betterprogramming.pub/how-to-upload-files-to-amazon-s3-from-nextjs-app-b7ef1909976b

As a junior developer, it's easier for me to proceed with a step by step
tutorial
